test/e2e/framework/framework.go

```diff
@@ -191,6 +201,52 @@ func (f *Framework) DeleteEtcdOperatorCompletely() error {
 	return nil
 }
 
+// SetupEtcdBackupOperator creates a etcd backup operator deployment with name as "etcd-backup-operator",
+// and it waits until the operator pod is ready.
+func (f *Framework) SetupEtcdBackupOperator() error {
+	cmd := []string{"/usr/local/bin/etcd-backup-operator"}
+	pod := &v1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   etcdBackupOperator,
+			Labels: map[string]string{"name": etcdBackupOperator},
+		},
+		Spec: v1.PodSpec{
+			Containers: []v1.Container{
+				{
+					Name:            etcdBackupOperator,
+					Image:           f.opImage,
+					ImagePullPolicy: v1.PullAlways,
+					Command:         cmd,
+					Env: []v1.EnvVar{
+						{
+							Name:      constants.EnvOperatorPodNamespace,
+							ValueFrom: &v1.EnvVarSource{FieldRef: &v1.ObjectFieldSelector{FieldPath: "metadata.namespace"}},
+						},
+						{
+							Name:      constants.EnvOperatorPodName,
+							ValueFrom: &v1.EnvVarSource{FieldRef: &v1.ObjectFieldSelector{FieldPath: "metadata.name"}},
+						},
+					},
+				},
+			},
+			RestartPolicy: v1.RestartPolicyNever,
+		},
+	}
+
+	p, err := k8sutil.CreateAndWaitPod(f.KubeClient, f.Namespace, pod, 60*time.Second)
+	if err != nil {
+		return err
+	}
+	logrus.Infof("etcd backup operator pod is running on node (%s)", p.Spec.NodeName)
+
+	return e2eutil.WaitUntilOperatorReady(f.KubeClient, f.Namespace, etcdBackupOperator)
```

Comment:
Just return nil.
There is no readiness check in backup operator yet.

Reply:
good to know!